lib/main_common.pm

```diff
@@ -296,13 +296,17 @@ sub load_slepos_tests {
 sub load_docker_tests {
     loadtest "console/docker";
     loadtest "console/runc";
-    # No package 'docker-compose' in SLE
+    # No package 'docker-compose, skopeo' in SLE<15
     if (!is_sle) {
         loadtest "console/docker_compose";
+        loadtest "console/skopeo";
```

Comment:
I would prefer just one mentioning in the if branch further below starting with line 307

Reply:
but then how it will run for opensuse tumbleweed? My intention is to run this for tumbleweed and SLE>15.

Reply:
Ok found a way to do it ;)